shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -867,23 +867,23 @@ async fn test_protocol_v4(#[case] driver: CassandraDriver) {
 #[case::cdrs(CdrsTokio)]
 #[tokio::test(flavor = "multi_thread")]
 #[serial]
-async fn test_protocol_v5(#[case] _driver: CassandraDriver) {
+async fn test_protocol_v5(#[case] driver: CassandraDriver) {
     let _docker_compose =
         DockerCompose::new("example-configs/cassandra-passthrough/docker-compose.yaml");
 
     let shotover = ShotoverProcessBuilder::new_with_topology(
-        "example-configs/cassandra-passthrough/topology-encode.yaml",
+        "example-configs/cassandra-passthrough/topology.yaml",
```

Comment:
why are we losing `-encode`?
Does it not support encoding?

Reply:
Mistake, I must've turned off during dev